examples/forward.hlb

```diff
@@ -0,0 +1,47 @@
+fs default() {
+	image "alpine" with option { resolve; }
+	run "docker version" with option {
+		mount dockerCli "/usr/local/bin"
+		forward "unix:///run/docker.sock" "/run/docker.sock"
+	}
+}
+
+fs buildDockerCli() {
+	image "golang:alpine" with option { resolve; }
+	run "apk add -U git bash coreutils gcc musl-dev"
+	env "CGO_ENABLED" "0"
+	env "DISABLE_WARN_OUTSIDE_CONTAINER" "1"
+	run "./scripts/build/binary" with option {
+		dir "/go/src/github.com/docker/cli"
+		mount fs { git "https://github.com/docker/cli.git" "v19.03.8"; } "/go/src/github.com/docker/cli"
+		mount fs { scratch; } "/go/src/github.com/docker/cli/build" as dockerCli
+	}
+}
+
+fs ssh() {
+	image "alpine"
+	run "apk add -U openssh-client"
+	mkdir "/root/.ssh" 0o700
+	run "ssh-keyscan github.com >> /root/.ssh/known_hosts"
+	run "ssh -q -T git@github.com || true" with option {
+		ignoreCache
+		ssh
+	}
+}
+
+fs nginx() {
+	image "alpine" with option { resolve; }
```

Comment:
minor nit, I dont think we need alpine here, I suspect the `dockerCli` base is sufficient since it is a static (no cgo) binary. Then you can change the `docker run` command to `./docker run ...`?

Reply:
I think since we want to make the nginx function more friendly, we still want to have a shell (not necessarily alpine). Maybe busybox is a better choice here?